test/draw/gpu/GpuSurfaceTest.ooc

```diff
@@ -128,9 +129,9 @@ GpuSurfaceTest: class extends Fixture {
 			}
 			gpuImage drawPoints(circlePoints)
 			rasterFromGpu := gpuImage toRaster()
-			expect(rasterFromGpu distance(correctImage), is equal to(0.0f))
+			expect(rasterFromGpu distance(correctImage), is less than(0.05f))
 			(correctImage, gpuImage, rasterFromGpu, trianglePoints, circlePoints) free()
-		})
+		})*/
```

Comment:
It is not a high priority since it is only used for debugging but we can have an issue for making a new shape test with thick lines.